input/fsh/valuesets/au-erequesting-request-status.fsh

```diff
@@ -0,0 +1,17 @@
+ValueSet: AUeRequestingRequestStatus
```

Comment:
Commenting here to get an inline comment. Please bind the new value set to ServiceRequest.status in AU eRequesting Diagnostic Request.